docs/android.md

```diff
@@ -78,9 +61,11 @@ Main Activity | Unity is loaded and is running in a separate Activity. Light gre
   <br><img src="images/android/openPackageManager.png" width='750px'>
   <br><img src="images/android/uguiPackages.png" width='750px'>
   <br><img src="images/android/unityUIPackages.png" width='750px'>
+- If the Build Output in Android Studio displays error messages like the image below, open SDK Manager and remove the build tool and then install it again.
```

Comment:
I think with newest addition in .androidlib, this error will never happen. Maybe remove